src/Traits/Deserializes.php

```diff
@@ -123,7 +123,7 @@ protected static function convertValueToDateTime(string $value): DateTime
     {
         foreach (static::$validDatetimeFormats as $validDatetimeFormat) {
             try {
-                $returnValue = DateTime::createFromFormat($validDatetimeFormat, $value);
+                $returnValue = DateTime::createFromFormat($validDatetimeFormat, $value, (str_ends_with($value, 'Z') ? new DateTimeZone('UTC') : null));
```

Comment:
I think it sorta makes sense to convert everything to UTC, because that way date strings (e.g., `Y-m-d`), which don't have a timezone at all, will default to UTC when converted into `DateTime` objects. Something in `DATE_ATOM` format (e.g., `2024-06-26T11:53:00+02:00`) will keep the correct timezone (`+02:00` in this case) even if `DateTime::createFromFormat()` is passed `new DateTimeZone('UTC')` as the timezone.

```php
$dt = DateTime::createFromFormat(
    DATE_ATOM,
    '2024-06-26T11:53:00+02:00',
    new DateTimeZone('UTC')
);
$dt->getTimeZone();  // -> DateTimeZone { timezone: +02:00 }
```

What do you think about that?

Reply:
Okay I see that adding the UTC timezone as parameter won't do any harm, I totally agree that adding that makes sense.

For returning a final DateTime object I suggest to return the time in the local timezone. The comfort of the function/library is not thinking about handling DateTime values. For further handling I need the DateTime in the local timezone.
So what do you think is better:
1. use the function to return the time in local timezone
2. keep returning UTC, then the idea would be to set a parameter in the library to define if local or UTC time is wanted. (there it would be best if you say how it fits best in the library structure)

Reply:
I always use UTC whenever possible, and in my experience most servers are configured to use UTC too. It's the standard that all other timezones are based off of. I'd prefer not to get into making timezones configurable in this library – I think that's something that the user can handle if they want to.

Reply:
I have some servers and they are all configured to the Berlin timezone where they are operated. 
I understand your approach to have server timezone to UTC and handle the app the rest, sadly not all apps are that way. 
For convenience reasons a library setting for the timezone would be great. It could be by default UTC so no-one has a different behaviour like now, and if needed it can be set different - like the user agent.

Reply:
Ok, sure, if you want to add a method to `SellingPartnerApi` to make it possible to set the default timezone, go for it :) please add some tests as well

Reply:
Adding a property to the main class (like user agent) is no problem. What are your thoughts and constructive concepts to access that new property from Deserializer Trait?

Reply:
Hmm, good point. There's no good way to do that. With that in mind, I think I'm going to stick to my original instinct here, which is that making timezones configurable feels outside the scope of this library. I'm going to leave it to the end user to convert dates to whatever timezone they'd like them to be in...it would take substantial work to support this.

Reply:
Ok good that I wasn't just stupid not to find a good way to get the property in the Trait ;-)
I changed the PR to always use UTC for Creation.